docs/examples/rx/03_listen_to_balance_change/index.js

```diff
@@ -22,7 +22,7 @@ async function main () {
       pairwise()
     )
     .subscribe((balance) => {
-      if (balance[0].eq('first')) {
+      if (balance[0] === 'first') {
```

Comment:
I introduced this bug before without testing the example, sorry :)

Reply:
It happens :)